nixos/doc/manual/from_md/release-notes/rl-2211.section.xml

```diff
@@ -125,6 +125,18 @@
           would in NixOS 22.05 and earlier.
         </para>
       </listitem>
+      <listitem>
```

Comment:
We no longer do xml files in nixos/* . Also please carry over the other changelog to 23.05